python/pipeline/meso.py

```diff
@@ -335,46 +333,34 @@ def _make_tuples(self, key):
         scan_filename = (experiment.Scan() & key).local_filenames_as_wildcard
         scan = scanreader.read_scan(scan_filename, dtype=np.float32)
 
-        for field_id in range(scan.num_fields):
```

Comment:
Why are we not applying `RasterCorrection` on all fields?

Reply:
Before corrections (raster and motion) were populated once per scan (and thus we needed to run a for loop over all fields, now I ran the populate once per field;  it makes it faster because all fields can be motion corrected in parallel (current bottleneck)